fleece/connexion.py

```diff
@@ -104,15 +106,22 @@ def call_api(self, event):
                     message=msg,
                 )
             elif 500 <= response.status_code < 600:
-                # This case is generally enountered if the API endpoint handler
-                # code does not conform to the contract dictated by the Swagger
-                # specification. This case will also trigger if the handler
-                # code explicitly returns a 5xx error.
-                self.logger.error(
-                    'Raising 5xx error',
-                    response=response_dict,
-                    http_status=response.status_code,
-                )
+                if response_dict['title'] == RESPONSE_CONTRACT_VIOLATION:
+                    # This case is generally enountered if the API endpoint
+                    # handler code does not conform to the contract dictated by
+                    # the Swagger specification.
+                    self.logger.error(
+                        RESPONSE_CONTRACT_VIOLATION,
+                        detail=response_dict['detail']
+                    )
```

Comment:
Yeah, that is I think a good solution, but shouldn't this new error handler return a correct 500 response object back to connexion? Or will that happen automatically if the handler doesn't return anything?